src/main/scala/common/parameters.scala

```diff
@@ -83,7 +83,9 @@ case class BoomCoreParams(
   useRVE: Boolean = false,
   useBPWatch: Boolean = false,
   clockGate: Boolean = false,
-  loadSliceMode: Boolean = false
+  loadSliceMode: Boolean = false,
```

Comment:
It might be worth having a separate substructure (like for the FPU) and use Some/None instead of many individual parameters here